smp/smp.c

```diff
@@ -59,6 +59,8 @@ void __noreturn ap_startup(void) {
     ap_callin = true;
     smp_wmb();
 
+    init_timers(cpu);
+
```

Comment:
We should exclude HPET and PIT for the APs, no? They're global resources that should be initialized only once -- by the BSP.

Reply:
Ugh, of course! Will fix.